client-tools/Shell/V8ShellFeature.cpp

```diff
@@ -310,9 +310,20 @@ void V8ShellFeature::copyInstallationFiles() {
       FileUtils::buildFilename("js", "node", "node_modules");
   std::string const nodeModulesPathVersioned = basics::FileUtils::buildFilename(
       "js", versionAppendix, "node", "node_modules");
+  std::string const uiNodeModulesPath =
+      FileUtils::buildFilename("js", "apps", "system", "_admin", "aardvark",
+                               "APP", "react", "node_modules");
   std::regex const binRegex("[/\\\\]\\.bin[/\\\\]", std::regex::ECMAScript);
 
-  auto filter = [&nodeModulesPath, &nodeModulesPathVersioned,
+  auto filterPath = [](std::string_view normalizedPath,
+                       std::string_view filterPath) -> bool {
+    return (!filterPath.empty() && normalizedPath.size() >= filterPath.size() &&
+            normalizedPath.substr(normalizedPath.size() - filterPath.size(),
+                                  filterPath.size()) == filterPath);
```

Comment:
Isn't this the same as `return !filterPath.empty() && normalizedPath.ends_with(filterPath);`?